### ISSUE TITLE
Default link should start with https

### ISSUE THREAD

**Issue by Xilonz** [soflyy/oxygen-bugs-and-features#961]
**Describe the bug**
When adding a link in the oxygen builder it starts with http:// by default, while currently, https is the standard.

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
N/a

**To Reproduce**
Add a link inside a text component

**Expected behavior**
The link to start with https:// by default

**What actually happens**
The link to starts with http://

**Screenshots**
N/a

**Desktop (please complete the following information):**
N/a

**Smartphone (please complete the following information):**
N/a

**Additional context**
-

**Comment by lucilecollignon** [comment]
I agree, this is giving me a headheck every time I launch a new website. Any news ? Or a way to fix it quickly ?

**Comment by pixluser** [comment]
Same problem, and it's even worse when you work with custom field, http:// is added no matter your build your link…
trying to do a wa.me link:
Here is my link in oxygen :
https://wa.me/[oxygen ct_sign_sha256='23c1f085c1f358fdda0c76ff075dcf5dffd1dbeb5ccbf6b7353b894336ce1353' data='custom_metabox_link' settings_path='whatsapp']
it render : https://wa.me/http://00336186188888

**Comment by arsh999cg** [comment]
@pixluser 

I've found a workaround until they provide us the solution :
Make the custom field as text field and put the full url there i.e. "https://wa.me/00336186188888"  and it will work fine.
